lint-staged.config.js

```diff
@@ -25,6 +25,10 @@ module.exports = {
     //
     // For that reason, we move the `--onlyChanged` flag next to it.
     'yarn lint:js --reporters=jest-silent-reporter --onlyChanged',
-    'tsc-files --noEmit',
```

Comment:
This si in purpose, and it's a wrapper around `tsc`. It's meant to be used with something like lint-staged.

What's the reason for changing that?

Reply:
I would have to go back and clone from scratch to recreate the issue, but this was changed because the `lint-staged` husky commit hook was failing for me after the initial install.  Changing the config to use this syntax made it so that the typecheck step ran without issue.

This issue/comment in the `lint-staged` github repo goes into more detail: https://github.com/okonet/lint-staged/issues/825#issuecomment-620018284

Reply:
Note that `tsc-files` is not the same as `tsc`. Please revert those changes as it was working fine before (we also have the same setup in other repos).
In any case, it's something not related with the scope of this PR, we can look at this separately.

Thanks